requirements-dev.txt

```diff
@@ -7,9 +7,10 @@ coverage==5.5,<5.6
 doc8>=0.8
 docformatter==1.4
 flake8>=3.8.3,<3.9
+flynt
 isort>5.5
 mock>4
-pylint>=2.7,<2.8
+pylint>=2.13.0 # Use at least version 2.13.0 to avoid security vulnerabilities (https://github.com/PyCQA/pylint/issues/5322)
```

Comment:
I fixed other comments, but I just have a remaining error with pylint, about a circular import. I think the error started appearing with the update of pylint's version, since it doesn't seem to be from this PR's code.

I get the error `R0401: Cyclic import (cowbird.constants -> cowbird.utils) (cyclic-import)`. Haven't found yet how to fix it, without refactoring everything from those files. Magpie seems to use a similar setup for those files, with similar circular imports and pylint disable comments.
I tried for fun to use pylint 2.14.2 on magpie, and I got 48 cyclic-import errors too :O

Anyway, I don't know yet how to fix it, or if there is a way to add a disable comment for this error (since it is not from a single line.) I can always disable the error entirely though
Let me know if you have any suggestions!

Reply:
I have checked  where the error are. Both for cowbird and magpie it is flagged for items imported within a function to avoid specifically those circular imports. Given that https://github.com/PyCQA/pylint/issues/850 is not implemented, I think the only thing we can do is disable R0401 project wise. Code "R" are for suggested refactors, so I'm not too concerned about them being ignored since they were handled correctly in the code.